src/cephadm/cephadm

```diff
@@ -1078,6 +1078,39 @@ def infer_fsid(func):
 
     return _infer_fsid
 
+def infer_config(func):
+    """
+    If we find a MON daemon, use the config from that container
+    """
+    @wraps(func)
+    def _infer_config():
+        if args.config:
+            logger.debug('Using specified config: %s' % args.config)
+            return func()
+        if not args.config and os.path.exists(SHELL_DEFAULT_CONF):
```

Comment:
maybe infer the config even if `SHELL_DEFAULT_CONF` exists?

Reply:
@jschmid1 I've changed it, so the new precedence order is:
1) `--config` option
2) config inferred from `MON`
3) default `/etc/ceph/ceph.conf` location